e2e/go.mod

```diff
@@ -1,21 +1,21 @@
 module e2e_test
 
-go 1.20
```

Comment:
Not sure if we should keep the go versions same for linode-cloud-controller-manager package and e2e tests. We bumped to 1.22 [here](https://github.com/linode/linode-cloud-controller-manager/blob/main/go.mod#L3), should we also change e2e to use 1.22?

Reply:
Done